### PR TITLE
Disable omitting class name from constructor

### PR DESCRIPTION
**Pull request by edwinkortman**
This PR is without issue because it comes from this discussion:
https://github.com/corgibytes/freshli-cli/pull/121#discussion_r924735640

### CODE REVIEW

File: Corgibytes.Freshli.Cli.Test/CommandOptions/ScanCommandOptionsTest.cs [hunk 1]
```diff
@@ -127,7 +127,7 @@ public ScanCommandOptionsTest(ITestOutputHelper output) : base(output)
         IList<OutputStrategyType> expectedOutput)
     {
         var cmBuilder = Program.CreateCommandLineBuilder();
-        Parser parser = new(cmBuilder.Command);
+        Parser parser = new Parser(cmBuilder.Command);
```
Comment (mscottford) [inline Corgibytes.Freshli.Cli.Test/CommandOptions/ScanCommandOptionsTest.cs:130]: This should become:

```suggestion
        var parser = new Parser(cmBuilder.Command);
```

**Review by mscottford — CHANGES_REQUESTED** [review]
I'm okay with the direction of this change because the `new(new(new(` that we've seen in a few places has been confusing. But I would like to see us use `var` in cases where there would otherwise be `TypeName value = new TypeName`.

File: Corgibytes.Freshli.Cli.Test/CommandOptions/ScanCommandOptionsTest.cs (hunk 1 above)
Comment (mscottford) [reply Corgibytes.Freshli.Cli.Test/CommandOptions/ScanCommandOptionsTest.cs]: I've addressed this in recent commits.

**Review by mscottford — APPROVED** [review]

**Review by donabp — APPROVED** [review]
everything looks good and clean, it's ready for a merge! :)

**Comment by mscottford** [comment]
recheck

**Comment by mscottford** [comment]
recheck

**Pull request merged**